run_tests.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Run tests for the Problem Builder XBlock
```

Comment:
Needs update - still says "Problem Builder"

Also the text below doesn't really apply since there are no selenium/bokchoy tests (yet).